### PR TITLE
Create SiliconValley_history.md

### PR DESCRIPTION
**Pull request by robymontyz**
Translated article about Silicon Valley History, suggested by Prof. De Martin.
It was an hard work.
TODO: fix line length

### CODE REVIEW

File: resources/SiliconValley_history.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+La spinta a costruire un’altra Silicon Valley può essere destinata a fallire, ma ciò non è necessariamente una cattiva notizia per i pianificatori territoriali di altrove. L'economia high-tech non è un gioco a somma zero. L'economia del settore tecnologico globale del XXI secolo è grande e complessa abbastanza per far sì che più regioni possano prosperare per decenni a venire - inclusa la Silicon Valley, se le minacce che deve affrontare verrano prese sul serio.
+
+
+
+
+*Leslie Berlin, 01/05/2015*
+
+Traduzione a cura di [Roberto Alessi](https://www.twitter.com/robymontyz) (m. S205065)  
+Articolo originale: https://medium.com/backchannel/why-silicon-valley-will-continue-to-rule-c0cbb441e22f
```
Comment (bassosimone) [inline resources/SiliconValley_history.md]: Could you please put this piece of information at the top?

**Comment by bassosimone** [comment]
I'd like to merge this, thanks! Have you checked whether you are allowed to translate the original document and, if so, under which license the document is?

**Comment by robymontyz** [comment]
The site the document come from says that author owns the rights but the terms are a bit strange (https://medium.com/policy/medium-terms-of-service-9db0094a1e0f). So I contacted the author to ask permissions and I am waiting for a reply.

**Comment by robymontyz** [comment]
I talked with the author and she is ok about the translation.

**Comment by bassosimone** [comment]
Awesome, thanks!

**Pull request merged**